pkg/cmd/ui/ui.go

```diff
@@ -108,6 +108,12 @@ func VerifyOutputPath(originalPath string, overwrite, outputPathOverridden, prom
 	return outputPath, doOverwrite
 }
 
+func VerifyGitPath(outputPath string) bool {
+	exists, err := ioutils.IsExisting(ioutils.NewFilesystem(), filepath.Join(outputPath, ".git"))
+	handleError(err)
+	return exists
+}
+
```

Comment:
Remove VerifySecretsPath()
change the ioutils func to use aferofs

Reply:
Will remove verifySecretsPath().
ioutils.NewFileSystem() returns an instance of aferofs. Are you suggesting to replace it?